### PR TITLE
feat: Added initial code of conduct

### PR DESCRIPTION
**Pull request by jvasallo**
- Missing contact of two as we work on consent

Headers might be off and maybe logo can be fixed but this has content parity of our old PDF code of Conduct. :)

### CODE REVIEW

File: src/assets/code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<center>Windy City DevFest 2019</center>
+========================================
+<img style="display: block; margin-left: auto; margin-right: auto;" width="300" height="300" src="./event-logo.svg">
+
+<center>Code of Conduct</center>
```
Comment (w9jds) [inline src/assets/code_of_conduct.md]: `<center>` doesn't work as intended (in github markdown), maybe try `<p style="text-align: center;">` instead.

File: src/assets/code_of_conduct.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+We can't follow up an anonymous report with you directly, but we will fully investigate it and take whatever action is necessary to prevent a recurrence.
+
+Personal Report
+---------------
+You can make a personal report by getting in contact with any of the event organizers. Event Organizers can be identified by special lanyards and/or special badges. If you wish, you can also use the anonymous form but provide your contact information at the end. In the event organizers are not able to be reached, please reach out to us directly by phone/email at:
+
+- General GDG/DevFest Organizer: – ([chicagogdg@gmail.com](mailto:chicagogdg@gmail.com)) 
+- Co-Organizer: Joel Vasallo – 773 318 8144 ([joelvasallo@gmail.com](mailto:joelvasallo@gmail.com)) 
+- WTM Lead/ Co-Organizer: <> – <> ([placeholder@exmaple.com](mailto:placeholder@example.com))
```
Comment (w9jds) [inline src/assets/code_of_conduct.md]: Probably want to put in Deana in here for us, worry about templating later.

**Review by w9jds — CHANGES_REQUESTED** [review]
Try taking a look at this in the Github viewer, some of the HTML doesn't show properly in this markdown format. 

Also, since we aren't going to template this quite yet, could you also fill in the information we care about and remove the stuff we don't.

File: src/assets/code_of_conduct.md (hunk 2 above)
Comment (jvasallo) [reply src/assets/code_of_conduct.md]: Done!

File: src/assets/code_of_conduct.md (hunk 1 above)
Comment (jvasallo) [reply src/assets/code_of_conduct.md]: Github Markdown apparently strips out style 😢

 Have to do it in a paragraph tag 🤢

File: src/assets/code_of_conduct.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
+<p align="center">
+  <img width="300" height="300" src="./event-logo.svg">
+</p>
+
+<center>
+
+  # Code of Conduct
+</center>
```
Comment (w9jds) [inline src/assets/code_of_conduct.md]: These `<center>` tags still don't do anything, update to use the same as the image.

**Review by w9jds — CHANGES_REQUESTED** [review]

File: src/assets/code_of_conduct.md (hunk 3 above)
Comment (jvasallo) [reply src/assets/code_of_conduct.md]: Let's see, it's not making it easy to play with headers unfortunately :(

**Review by w9jds — APPROVED** [review]

**Pull request merged**